backend/src/mailer/sendAccountMultiRegistrationEmail.test.ts

```diff
@@ -0,0 +1,30 @@
+import { sendAccountMultiRegistrationEmail } from './sendAccountMultiRegistrationEmail'
+import { sendEMail } from './sendEMail'
+
+jest.mock('./sendEMail', () => {
+  return {
+    __esModule: true,
+    sendEMail: jest.fn(),
+  }
+})
+
+describe('sendAccountMultiRegistrationEmail', () => {
+  beforeEach(async () => {
+    await sendAccountMultiRegistrationEmail({
+      firstName: 'Peter',
+      lastName: 'Lustig',
+      email: 'peter@lustig.de',
+    })
+  })
+
+  it('calls sendEMail', () => {
+    expect(sendEMail).toBeCalledWith({
+      to: `Peter Lustig <peter@lustig.de>`,
+      subject: 'Gradido: Erneuter Registrierungsversuch mit deiner E-Mail',
+      text:
+        expect.stringContaining('Hallo Peter Lustig') &&
+        expect.stringContaining('http://localhost/forgot-password') &&
+        expect.stringContaining('https://gradido.net/de/contact/'),
```

Comment:
at this point, you could also use the urls from the config, since they have been created.

Reply:
Done 👍🏼